docs/Troubleshooting.md

```diff
@@ -52,15 +72,13 @@ function todos(state = [], action) {
         }
       ]
     case 'COMPLETE_TODO':
-      // Return a new array
-      return [
-        ...state.slice(0, action.index),
-        // Copy the object before mutating
-        Object.assign({}, state[action.index], {
-          completed: true
-        }),
-        ...state.slice(action.index + 1)
-      ]
+      // Copy elements to a new array
+      state = [...state]
```

Comment:
This is problematic. We’ve seen people doing this and then mutating something deep inside the objects in array, thinking `state = [...state]` saved them. I know you have `Object.assign()` below but people will skip that and remember just `state = [...state]`.